test-cases/gutenberg/file.md

```diff
@@ -0,0 +1,219 @@
+
+# File Block - Test Cases
+##### TC001
+
+**Upload completes successfully from the device library**
+
+**Steps**
+- Tap on Choose a file on the Media Placeholder. 
+- Tap option "Choose from device" and choose a file. 
+
+**Expected Behavior**
+- Expect to see the upload indicator and the Download button should be dimmed.
+- Expect to not see a link or replace buttons in the toolbar.
+
+![Upload pending](../resources/local-file-upload-pending.png)
+- Expect to see file's name and an active Download button once the upload is complete.  
+- Expect the link and replace buttons to appear in the toolbar
+
+![Upload successful](../resources/local-file-upload-successful.png)
+
+--------------------------------------------------------------------------------
+
+##### TC002
+
+**Upload completes successfully from the WordPress Media library**
```

Comment:
Nitpick: Adding from WordPress Media Library is technically not an Upload. It's just inserted.

Reply:
Fixed in bfad517 along with placeholder changes we had discussed.